src/components/select-field/BaseSelectField.js

```diff
@@ -396,6 +415,7 @@ class BaseSelectField extends React.Component<Props, State> {
                             role="listbox"
                             // preventDefault on mousedown so blur doesn't happen before click
                             onMouseDown={event => event.preventDefault()}
+                            style={{ maxHeight: `calc(100vh - ${this.offsetTopValue}px)` }}
```

Comment:
I suspect some instances of these select fields may need to be constrained to one or their parent elements rather than to the document via `vh`.

Reply:
In that case, it might be better to set a defined `max-height` via CSS and remove the JS calculations in this component. Thoughts?

Reply:
Yeah, possibly. It seems like a reasonable compromise on the behavior and would be a much simpler implementation.